frontend/packages/console-shared/src/components/shortcuts/Shortcut.tsx

```diff
@@ -70,6 +74,11 @@ const Shortcut: React.FC<ShortcutProps> = ({
             <MouseIcon /> {t('console-shared~Right click')}
           </ShortcutCommand>
         )}
+        {customTitle && (
+          <ShortcutCommand>
+            {icon || <MouseIcon />} {customTitle}
+          </ShortcutCommand>
+        )}
```

Comment:
Component API seems to handle every shortcut individually. Specifying `dragNDrop` seems like a better choice.

```suggestion
        {dragNdrop && (
          <ShortcutCommand>
            <MouseIcon /> Drag + Drop
          </ShortcutCommand>
        )}
```

Reply:
I was in two minds while creating this